### ISSUE TITLE
img missing alt attribute

### ISSUE THREAD

**Issue by karlmacklin** [bombsimon/det-blev-ingen-cd.com#4]
The main image on the site is missing its alt attribute.

Not only is this detrimental to SEO visibility, it is not very user friendly from a WCAG perspective.

Consider upgrading to more common html5 approaches, perhaps the picture tag or using figcaption to provide a text alternative to the image, as the text in the image itself is very hard to read (see issue #3).
